test/performance/range/gap_decorator_helper.hpp

```diff
@@ -0,0 +1,162 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2019, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2019, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE
+// -----------------------------------------------------------------------------------------------------
+
+#include <chrono>
+#include <cmath>
+#include <cstring>
+#include <random>
+#include <utility>
+
+#include <benchmark/benchmark.h>
+
+#include <seqan3/alignment/aligned_sequence/aligned_sequence_concept.hpp>
+#include <seqan3/alignment/exception.hpp>
+#include <seqan3/alphabet/all.hpp>
+#include <seqan3/range/decorator/all.hpp>
+#include <seqan3/std/ranges>
+
+using namespace seqan3;
+
+#define SEQ_LEN_LONG 1<<18
+#define SEQ_LEN_SHORT 1<<12
+
+/*
+ * Apply benchmarks with custom ranges for grid parameters sequence length and gap
+ * proportions. When setting the SEQAN3_LONG_TESTS macro, the longer benchmark
+ * version will be executed.
+ *
+ *  | flag                | sequence lengths | gap proportions
+ *  | --------------------|------------------|---------------------
+ *  | !SEQAN3_LONG_TESTS  |  1 << [4:2:12]   | [1, 5, 50] %
+ *  | SEQAN3_LONG_TESTS   |  1 << [4:2:18]   | [1, 5, 25, 50, 75] %
+ */
+static void CustomArguments(benchmark::internal::Benchmark* b) {
```

Comment:
```suggestion
void custom_arguments(benchmark::internal::Benchmark * b)
{
```

no static, camel_case and style guide